spring-auto-restdocs-docs/index.adoc

```diff
@@ -44,6 +44,9 @@ Does it work with REST Assured tests?::
   Not yet. We may add support for REST Assured in the future, but your PR is also welcome.
 Is Jackson required for automatic field documentation?::
   Yes, this project only includes a Jackson visitor so far.
+Can multiple directories be configured with `javadocJsonDir`?::
```

Comment:
rephrase to more generic sentence, like "Can I use auto-restdocs in multi-module projects"?
- then Explain what kind of support there is for multi-module projects.